modules/student.rb

```diff
@@ -1,11 +1,20 @@
-require '../ibrary'
+require_relative '../person'
+require_relative './classroom'
+
 class Student < Person
-  def initialize(name, age, parent_permission, classroom)
-    super(name, age, parent_permission)
+  attr_reader :classroom
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] There are some errors in your app this in your app as you can see in the screenshot below ⬇️  Please be sure to fix them in order to make your app able to create a student and also a rental to make it works properly 👍 

![Capture](https://user-images.githubusercontent.com/6615394/225904615-d306c4fb-5c5b-40a6-bc83-20bb84f3c6e2.PNG)